### PR TITLE
Setup test configuration

### PR DESCRIPTION
**Pull request by Siegrift**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ function getServiceData(apiName: string, beaconName: string, chain: string) {
 
   const goBeaconData = goSync(() => require(`@api3dao/operations/data/apis/${apiName}/beacons/${beaconName}`));
   if (!isGoSuccess(goBeaconData)) throw new Error(`Service data does not exist.`);
-  const beaconData = JSON.parse(goBeaconData.data).toString();
+  const beaconData = goBeaconData.data;
```
Comment (Siegrift) [inline src/index.ts:23]: This was a bug before - since after the data is imported it is already parsed into JS object - not JSON text.

**Comment by dcroote** [comment]
It looks like the target branch of this is `repo-structure` rather than `main`?

**Comment by Siegrift** [comment]
@dcroote yeah, but it's strange - when the repo-structure branch was merged to main, github should change the base automatically.

File: package.json [hunk 2]
```diff
@@ -14,12 +14,15 @@
...
   ],
   "scripts": {
-    "lint": "yarn run prettier:check && yarn run lint:eslint",
     "lint:eslint": "eslint . --ext .js,.ts",
+    "lint": "yarn run prettier:check && yarn run lint:eslint",
     "prettier:check": "prettier --check \"./**/*.{js,ts,md,json}\" --loglevel silent",
-    "prettier:write": "prettier --write \"./**/*.{js,ts,md,json}\""
+    "prettier:write": "prettier --write \"./**/*.{js,ts,md,json}\"",
+    "test": "jest",
+    "tsc": "tsc --project tsconfig.json"
```
Comment (acenolaza) [inline package.json:22]: Should this one be "compile" or "build" instead?

File: src/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import services from '.';
+
+describe('getServiceData', () => {
+  it('works', () => {
```
Comment (acenolaza) [inline src/index.test.ts:4]:
```suggestion
  it('returns Amberdata service data', () => {
```

**Review by acenolaza — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Siegrift) [reply package.json:22]: Could be, but that one is mainly for linting the code. I'll create a proper build script in next PR.

**Pull request merged**